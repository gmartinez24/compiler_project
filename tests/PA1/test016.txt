// Arvind Murthy
// Testing whitespace examples given in project description
= ==
====
/ /
mainif
main if
abc def