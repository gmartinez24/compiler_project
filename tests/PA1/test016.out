ASSIGN
EQUAL_TO
EQUAL_TO
EQUAL_TO
DIV
DIV
IDENT	mainif
MAIN
IF
IDENT	abc
IDENT	def
EOF